test/views/github-dotcom-markdown.test.js

```diff
@@ -84,31 +87,118 @@ describe('GithubDotcomMarkdown', function() {
   });
 
   it('registers command handlers', function() {
-    const openIssueishLinkInNewTab = sinon.stub();
-    const openLinkInBrowser = sinon.stub();
-    const switchToIssueish = sinon.stub();
+    const openIssueishLinkInNewTabStub = sinon.stub();
+    const openLinkInBrowserStub = sinon.stub();
+    const switchToIssueishStub = sinon.stub();
 
     const wrapper = mount(buildApp({
       html: `
         <p>
           <a data-url="https://github.com/aaa/bbb/issue/123" href="https://github.com/aaa/bbb/issue/123">#123</a>
         </p>
       `,
-      openIssueishLinkInNewTab,
-      openLinkInBrowser,
-      switchToIssueish,
+      openIssueishLinkInNewTab: openIssueishLinkInNewTabStub,
+      openLinkInBrowser: openLinkInBrowserStub,
+      switchToIssueish: switchToIssueishStub,
     }));
 
     const link = wrapper.getDOMNode().querySelector('a');
     const href = 'https://github.com/aaa/bbb/issue/123';
 
     atom.commands.dispatch(link, 'github:open-link-in-new-tab');
-    assert.isTrue(openIssueishLinkInNewTab.calledWith(href));
+    assert.isTrue(openIssueishLinkInNewTabStub.calledWith(href));
 
     atom.commands.dispatch(link, 'github:open-link-in-this-tab');
-    assert.isTrue(switchToIssueish.calledWith('aaa', 'bbb', 123));
+    assert.isTrue(switchToIssueishStub.calledWith('aaa', 'bbb', 123));
 
     atom.commands.dispatch(link, 'github:open-link-in-browser');
-    assert.isTrue(openLinkInBrowser.calledWith(href));
+    assert.isTrue(openLinkInBrowserStub.calledWith(href));
+  });
+
+  describe('opening issueish links', function() {
```

Comment:
would it make sense to add a unit test here to assert that if the link opening fails, we don't record an event?

Reply:
done! 👍